.betterer.eslint.config.js

```diff
@@ -122,11 +122,11 @@ module.exports = [
         'error',
         {
           selector: 'Identifier[name=localStorage]',
-          message: 'Direct usage of localStorage is not allowed. import store from app/core/store instead.',
+          message: 'Direct usage of localStorage is not allowed. import store from app/core/store in core and Store from @grafana/data in packages (grafana-data, grafana-runtime, grafana-prometheus, grafana-ui, etc.)',
         },
         {
           selector: 'MemberExpression[object.name=localStorage]',
-          message: 'Direct usage of localStorage is not allowed. import store from app/core/store instead.',
+          message: 'Direct usage of localStorage is not allowed. import store from app/core/store in core and Store from @grafana/data in packages (grafana-data, grafana-runtime, grafana-prometheus, grafana-ui, etc.)',
```

Comment:
(you could set up two different applications of the rule if you really wanted with different messages, but as its betterer I wouldn't necessarily bother 😂 )